src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -1498,14 +1499,14 @@ def shift(
     @property
     def index(self) -> Union["pd.Index", native_pd.MultiIndex]:
         """
-        Get pandas index. The method eagerly pulls the values from Snowflake because index requires the values to be
-        filled
+        Get index. If multi-index, the method eagerly pulls the values from Snowflake because index requires the values to be
+        filled and returns a pandas multi index. If not multi-index, create a modin index and pass it self
```

Comment:
Nit:
```suggestion
        Get index. If MultiIndex, the method eagerly pulls the values from Snowflake because index requires the values to be
        filled and returns a pandas MultiIndex. If not MultiIndex, create a modin index and pass it self
```